### PR TITLE
Dummy scenario with three rovers

### PR DESCRIPTION
**Pull request by nmarshak1337**
We can now play with three rovers in a dummy scenario (choose scenario_dummy, which comes before mainMenuScenario).

### CODE REVIEW

File: index.vwf.config.yaml [hunk 1]
```diff
@@ -23,6 +23,6 @@ view:
   vwf/view/threejs: "#vwf-root"
   vwf/view/blockly: { "blocklyPath": './source/blockly/', "toolbox": 'toolbox' }
   vwf/view/sound:
-  # vwf/view/editor:
+  vwf/view/editor:
```
Comment (BrettASwift) [inline index.vwf.config.yaml]: Re-disable this. We never want the editor enabled on a public version of the game.

**Comment by nmarshak1337** [comment]
Now we have multiple rovers in a dummy scenario, with Spencer's additions (procedures and variables) incorporated, along with all of our original blocks. 

Known Issues:
- This breaks tracing, but Kevin Dill gave me the goahead. 
- In the dummy scenario, switch to Rover 2, then back to Rover 1, for the "drive" block to NOT be grayed out.

**Comment by kadst43** [comment]
Just to confirm... all blocks should be available, not just the drive block, right?

**Comment by nmarshak1337** [comment]
@kadst43 Per the discussion we just had, I need to bolt on the graph tool, plus add a few more things to the toolbox (everything in the graph tool plus y=, also make sure that everything that we use in the game actually goes into the toolbox.

**Comment by kadst43** [comment]
And a kitchen sink.  Don't forget the kitchen sink. ;)

You might close this PR, and then re-open when it's ready...

**Pull request closed without merging**